RockLib.Messaging.Http/HttpClientSender.cs

```diff
@@ -13,36 +13,75 @@ namespace RockLib.Messaging.Http
     /// An implementation of <see cref="ISender" /> that sends messages with an
     /// <see cref="HttpClient"/>.
     /// </summary>
+#pragma warning disable CS1069
```

Comment:
I know I haven't been consistent with this, but I'm thinking that if we put in a `#pragma`, we should include the text after it that VS does, and also include a comment with it that explains why we're ignoring it. That may be too much, but I'm wondering what we think about that.

Reply:
That was from some troubleshooting, and isn't actually needed.  I've gone back and removed it.  However, I agree, comments on these is probably a good idea.